app/static/event/eventview.css

```diff
@@ -50,10 +50,6 @@ div.event_info_row,
 	margin-block-end: 0.2em;
 }
 
-.title {
```

Comment:
No one uses that?

Reply:
It was used in view_event_details_tab.html and nowhere else.
The event color was set to blue by default in class .title.
I removed the class and switched it with a class for each color, according to the events color in the db.